Quoridor.Model/Cell.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Quoridor.Model
+{
+	internal class Cell : IElement
+	{
```

Comment:
Can we add a field for id, for better implementation of search path algorithm?
```suggestion
	{
	    public int Id { get; private set; }
```